src/main.rs

```diff
@@ -128,10 +131,18 @@ pub fn list_languages(assets: &HighlightingAssets, term_width: usize) {
     }
 }
 
-pub fn list_themes(assets: &HighlightingAssets) {
+pub fn list_themes(assets: &HighlightingAssets, cfg: &Config) {
     let themes = &assets.theme_set.themes;
+    let mut config = cfg.clone();
+    let mut style = HashSet::new();
+    style.insert(OutputComponent::Plain);
+    config.files = vec![Some("assets/theme_preview.rs")];
```

Comment:
I missed that previously, but that's obviously not going to work if we are not in `bat`s source folder.

Reply:
I think I have a solution for this